app/Board.php

```diff
@@ -13,4 +14,9 @@ public function user()
     {
         return $this->belongsTo(User::class)->select(['id', 'name']);
     }
+
+    public function idea()
```

Comment:
It would be better if this is named `ideas` (plural) so that it is easier to distinguish if it is a single record or multiple records from the relationship method.